src/sentry/static/sentry/app/views/alerts/rules/index.tsx

```diff
@@ -210,11 +206,11 @@ class AlertRulesList extends AsyncComponent<Props, State & AsyncComponent['state
       flatten(ruleList?.map(({projects}) => projects))
     );
 
-    const sort = {
-      ...DEFAULT_SORT,
+    const sort: {asc: boolean; field: 'date_added' | 'name'} = {
       asc: query.asc === '1',
-      // Currently only supported sorting field is 'date_added'
+      field: query.sort || 'date_added',
     };
+    const {cursor: _cursor, page: _page, ...currentQuery} = query;
```

Comment:
this is the only change in the frontend since the last PR that was reverted. On sort, removes the cursor/page